DuckDuckGo/AppDelegate.swift

```diff
@@ -66,8 +66,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationDidBecomeActive(_ application: UIApplication) {
         guard !testing else { return }
         
-        Pixel.fire(pixel: .appLaunch)
-        StatisticsLoader.shared.load()
+        StatisticsLoader.shared.load {
+            StatisticsLoader.shared.refreshAppRetentionAtb()
+            Pixel.fire(pixel: .appLaunch)
```

Comment:
There were questions asked about whether this was the right place to fire this pixel (ie is our app launch logic correct?).  Does that apply here? No one really cares about the pixels which is why we never prioritised fixing it,  but it could be more of an issue for the stats which we'll end up depending on more.

Reply:
This is just a small fix I put in place as I noticed that there is a race condition where the pixel may fire before the atb request making a blank atb more likely, the pixel logic is otherwise unchanged by this work. There is however another task to look at the launch pixel timing across both apps.

Reply:
Fair enough, but doesn't the refreshAppRetentionAtb happen async and could that mean the pixel still has a blank atb?

Did you mean to put it in the completion handler for the refresh call?

Reply:
The pixel only needs the initial atb to be loaded, it doesn't care about the retention atb so the code is as intended.